filebeat/input/docker/input.go

```diff
@@ -92,6 +111,11 @@ func NewInput(
 		return nil, errors.Wrap(err, "update input config")
 	}
 
+	// Set symlinks to true as CRI-O paths could point to symlinks instead of the actual path.
+	if err := cfg.SetBool("symlinks", -1, true); err != nil {
```

Comment:
Should this option be set only if paths are used? (I guess the answer is that it also works if the files are not symlinks as talked offline :slightly_smiling_face: )